app/Models/Extensions/AlbumBuilder.php

```diff
@@ -92,7 +93,7 @@ public function getModels($columns = ['*']): array
 			]);
 		}
 
-		return parent::getModels($columns);
+		return parent::getModels($columns); // @phpstan-ignore-line
```

Comment:
Add a comment with an explanation.

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1336/commits/3817459b78f214d434e1cf8b6cb86820b4631eae.